src/components/NFTPage/Recommendation.vue

```diff
@@ -153,59 +238,24 @@ export default {
     swiper() {
       return this.$refs.recommendationSwiper.$swiper;
     },
-    nftFeaturedBooks() {
-      return this.bookstoreListItemForLandingPage.length
-        ? this.bookstoreListItemForLandingPage
-        : [...DEFAULT_RECOMMENDATIONS_LIST.BOOK.map(nft => ({ classId: nft }))];
-    },
-    normalizedList() {
-      const normalizedRecommendedList = [...this.recommendedList];
-      const displayItemCount = Number(this.displayItemCount);
-      if (this.recommendedList.length < displayItemCount) {
-        let index = 0;
-        let sourceData = this.isBookNft
-          ? this.nftFeaturedBooks
-          : this.nftFeaturedWNFT;
-        // TODO: remove filter after recommendation support collection
-        sourceData = sourceData.filter(nft =>
-          // TODO: find out and fix classId undefined reason
-          nft.classId?.startsWith('likenft1')
-        );
-        while (
-          normalizedRecommendedList.length < displayItemCount &&
-          index < sourceData.length
-        ) {
-          const itemToAdd = sourceData[index];
-          const alreadyExists = normalizedRecommendedList.some(
-            item => item.classId === itemToAdd.classId
-          );
-
-          if (!alreadyExists) {
-            normalizedRecommendedList.push(itemToAdd);
-          }
-          index += 1;
-        }
-      }
-      return normalizedRecommendedList;
-    },
   },
-  watch: {
-    async recommendedList(list) {
-      if (this.isBookNft && list.length < +this.displayItemCount) {
-        try {
-          await this.fetchBookstoreItemsFromCMSForLandingPage();
-        } catch (error) {
-          console.error(error);
-        }
+  mounted() {
+    if (this.isAuthorSpecific) {
+      this.lazyFetchCreatedNFTClassesByAddress(this.productOwner);
+    }
```

Comment:
```suggestion
    if (this. productOwner) {
      this.lazyFetchCreatedNFTClassesByAddress(this.productOwner);
    }
```
Seems more straight forward, if we can confirm `productOwner` is not set in this case

Reply:
[♻️ Simplify logic for determining default display](https://github.com/likecoin/liker-land/pull/1863/commits/a66dbc9fbf89afb0dc08753c0774612b6dc4ba0c)